src/nationalcareers_toolkit/assets/src/frontend/js/compui/apps/dysac.js

```diff
@@ -12,7 +12,11 @@
 //    results.long();
 //}
 
+
+
 $(document).ready(function () {
+    ShowErrorInPageTitle('dysac-validation-summary');
```

Comment:
You will need to prefix this call with:

compUiShell.validation

Reply:
changed